spec/util/puppet_agent_mgr_spec.rb

```diff
@@ -327,20 +327,28 @@ module MCollective::Util
         end
 
         it "should fail when a daemon is idling " \
-           "and tags, environment or noop is specified" do
+           "and tags, skip_tags, environment or noop is specified" do
```

Comment:
It happens that tags test was missing, so I added.  I reordered the existing tests so that the tests order match the test description.